### PR TITLE
Acl inventory

### PR DESCRIPTION
**Pull request by haiders5**
Adding ACL Inventory workflow to retrieve all IP addresses associated to an ACL and download the list in csv format

### CODE REVIEW

**Comment by haiders5** [comment]
Adding two workflows to Community

File: Community/create_a_record/README.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Copyright 2019 BlueCat Networks (USA) Inc. and its affiliates
```
Comment (steven0784) [inline Community/create_a_record/README.txt]: so it seems like you already have a pull request for create_a_record in https://github.com/bluecatlabs/gateway-workflows/pull/19 , not sure why this is in this pull request

File: Community/acl_inventory/acl_inventory_page.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        g.user.logger.error('%s' % e, msg_type=g.user.logger.EXCEPTION)
+        flash('Unable to retrieve the selected ACL - Please try again')
+        return redirect(url_for('acl_inventoryacl_inventory_acl_inventory_page'))
+
+    # Retrieve IP addresses associated with ACL
+    try:
+        value_list = acl.get_property('aclValues').split(',')
+    except ValueError as e:
+        g.user.logger.error('%s' % e, msg_type=g.user.logger.EXCEPTION)
+        flash('An unexpected error occurred')
```
[the remark this answers is not in the document]
Comment (haiders5) [reply Community/acl_inventory/acl_inventory_page.py]: Thats only if it goes to the exception catcher in the header files. I dont want it to do that here.

**Comment by steven0784** [comment]
Please resolve the conflict

**Comment by haiders5** [comment]
Going to re-commit

**Pull request closed without merging**